aws/resource_aws_batch_compute_environment.go

```diff
@@ -164,7 +164,8 @@ func resourceAwsBatchComputeEnvironment() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{batch.CEStateEnabled, batch.CEStateDisabled}, true),
 				Default:      batch.CEStateEnabled,
 			},
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
```

Comment:
There's also a `tags` field in the `compute_resources` block that is (according to the documentation):

> Key-value pair tags to be applied to EC2 resources that are launched in the compute environment. 

I'm guessing that adding `tags_all` here would mean complex changes to a variant of `SetTagsDiff`, so maybe we want to open a tracking issue for this case.

Reply:
The initial rollout of default tags is intended for resource-level tags only.